tests/mutation_test.cc

```diff
@@ -1320,6 +1320,99 @@ SEASTAR_THREAD_TEST_CASE(test_mutation_upgrade_type_change) {
     assert_that(m).is_equal_to(m2);
 }
 
+SEASTAR_THREAD_TEST_CASE(test_row_marker) {
```

Comment:
I would really appreciate a comment above the test explaining what it's testing, and why. If it served you to reproduce an issue with a number, you can say it reproduces issue #1234, for example. Or if this test is just a general thing, and just one specific line (e.g., the test for t2 - the equality case) reproduces issue #1234 you can say it there.

Reply:
OK, I added some description above the test.